muspinsim/muspinsim.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="muspinsim" name="Simulate muon experiment spin dynamics (muspinsim)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip install git+https://github.com/muon-spectroscopy-computational-project/muspinsim.git &&
+        ln -s $mu_sim $mu_sim.name &&
+        ln -s $mu_exp_in $mu_exp_in.name &&
+        ## grab 'name' field from input file, if it doesn't exist, set name as 'muspinsim'
+        mu_out=\$(grep -A1 '^name' $mu_sim.name | grep -v 'name' | sed -e 's/^[ \t]*//') &&
+        mu_out=\${mu_out:-"muspinsim"}.dat &&
+        log_out=\$(echo $mu_sim.name | cut -d '.' -f1).log &&
+        ## if fitting data required, change filepath in input file to point to symlink
+        if grep '^fitting_data' $mu_sim; then sed -iE "/^fitting_data/{n;s/([^)]*)/(\"$mu_exp_in.name\")/g}" $mu_sim.name; fi &&
+        muspinsim $mu_sim.name &&
+        mv \$mu_out mu_out.dat &&
+        mv \$log_out log_out.log
+    ]]></command>
+    <inputs>
+        <param type="data" name="mu_sim" label="input file containing simulation parameters"/>
+        <param type="data" name="mu_exp_in" optional="true" label="Experiment data to fit (.dat) (Optional)" />
+    </inputs>
+    <outputs>
+        <data label="muspinsim results for $mu_sim.name" name="mu_out" format="txt" from_work_dir="mu_out.dat" />
+        <data label="muspinsim log for $mu_sim.name" name="log_out" format="txt" from_work_dir="log_out.log" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="mu_sim" value="hfine.in" ftype="txt" />
+            <output name="mu_out" file="hfine.dat" ftype="txt" compare="sim_size" lines_diff="20" />
+            <output name="log_out" file="hfine.log" ftype="txt" compare="sim_size" delta="100" />
+        </test>
+        <test>
+            <param name="mu_sim" value="hfine_powder.in" ftype="txt" />
+            <output name="mu_out" file="hfine_powder.dat" ftype="txt" compare="sim_size" delta="20" />
+            <output name="log_out" file="hfine_powder.log" ftype="txt" compare="sim_size" delta="100" />
+        </test>
+        <test>
+            <param name="mu_sim" value="fitting.in" ftype="txt" />
+            <param name="mu_exp_in" value="fitting_input.dat" ftype="txt" />
+            <output name="mu_out" file="fitting_data.dat" ftype="txt" compare="sim_size" delta="20" />
+            <output name="log_out" file="fitting.log" ftype="txt" compare="sim_size" delta="100" />
+        </test>
+    </tests>
+    <help><![CDATA[
+        MuSpinSim is a program designed to carry out spin dynamics calculations
+
+        Tool requires an input (.in) file which describes system and experiment details
```

Comment:
Beware - this is RST format and two newlines are needed to create a line break in the rendered document.
![image](https://user-images.githubusercontent.com/20683271/163134111-30719df5-2351-4cda-91c6-5e7b7ff615de.png)
```suggestion
        Tool requires an input (.in) file which describes system and experiment details.
```